llms/mlx_lm/server.py

```diff
@@ -540,10 +553,17 @@ def main():
 
     # Building tokenizer_config
     tokenizer_config = {"trust_remote_code": True if args.trust_remote_code else None}
+    if args.chat_template:
+        tokenizer_config['chat_template'] = args.chat_template
 
     model, tokenizer = load(
         args.model, adapter_path=args.adapter_path, tokenizer_config=tokenizer_config
     )
+
+    if args.use_default_chat_template:
```

Comment:
Should it be an error to specify a chat-template and `use-default-chat-template`?

Reply:
Sounds good!